lib/operations/mongo_client_ops.js

```diff
@@ -416,6 +417,43 @@ function createTopology(mongoClient, topologyType, options, callback) {
     }
 
     assignTopology(mongoClient, newTopology);
+    if (options.autoEncryption == null) {
+      callback(null, newTopology);
+      return;
+    }
+
+    // setup for client side encryption
+    let AutoEncrypter;
+    try {
+      AutoEncrypter = require('mongodb-client-encryption').AutoEncrypter;
```

Comment:
request: can we move this out to a separate location, similar to our logic for pulling in `kerberos`?

Reply:
I considered this, but that approach eventually unrolls into two try/catch statements since we need to pass errors to the callback here. It would look like:
```
function retrieveAutoEncrypter() {
  let AutoEncrypter;

  try {
    AutoEncrypter = require('mongodb-client-encryption').AutoEncrypter;
  } catch (err) {
    if (err.code === 'MODULE_NOT_FOUND') {
      throw new Error('Auto-encryption requested, but the module is not installed. Please add `mongodb-client-encryption` as a dependency of your project');
    }

    throw err;
  }

  return AutoEncrypter;
}


...

let AutoEncrypter;
try {
   AutoEncrypter = retrieveAutoEncrypter();
} catch (err) {
  callback(err, null);
}
```